src/main/java/duke/task/EventTask.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import duke.exceptions.DukeException;
+
+/**
+ * Class for event tasks
+ */
+public class EventTask extends Task {
+
+    protected String start_time, end_time;
+
+    /**
+     * Constructor for EventTask
+     *
+     * @param name Name of the event
+     * @param start_time Starting time for the task
+     * @param end_time Ending time for the task
+     * @param file_format File format for the task
+     *
+     */
+    public EventTask(String name, String start_time, String end_time, String file_format) {
+        super(name, Task.Type.E, file_format);
+        this.start_time = start_time;
+        this.end_time = end_time;
+
+    }
+
+    /**
+     * String representation of EventTask
+     *
+     * @return String representation of EventTask
+     */
+    @Override
+    public String toString() {
```

Comment:
Implement the toString() method in your Task.java.
This way you can call it by super.toString() in each of the subclasses of Task.java.
This is helpful since each of the subclasses toString() methods have commonalities in the string it prints.
For reference, this is mine:
 String check = isDone ? "[X] " : "[ ] ";
 return check + this.name;